.github/workflows/tests.yml

```diff
@@ -207,7 +207,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest-16-cores, macos-latest, windows-latest-8-cores]
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
I'm tipping that `ubuntu-latest` is running out of memory for the multimodal example test